src/_data/LiveData.js

```diff
@@ -48,18 +48,22 @@ export const initUnauthorized = () => {
     getVideosFromPlayList().then(videos => api.events.emit('videos', videos));
 };
 
-export const initAuthorized = () => {
+export const initAuthorized = (authData) => {
+    const isVirtual = authData.authorize.loginid.startsWith('VR');
```

Comment:
You don't need to put on top now. const & let are not hoisted, that is old school ;)